.github/CODEOWNERS

```diff
@@ -0,0 +1,76 @@
+# Describes which individuals/teams are reponsible for which code in the
+# repository.
+#
+# See: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+/.cargo                             @benesch
+/.config                            @benesch
+/.devcontainer                      @mjibson
+/.github                            @benesch
+/bin
+/ci                                 @benesch
+/doc/user                           @MaterializeInc/DevEx
+/doc/developer/reference/compute    @MaterializeInc/compute
+/doc/developer/reference/storage    @MaterializeInc/storage
+/ci                                 @MaterializeInc/qa
+/src/adapter                        @MaterializeInc/surfaces
+/src/alloc                          @benesch
+/src/audit-log                      @MaterializeInc/surfaces
+/src/avro                           @umanwizard
+/src/avro-derive                    @umanwizard
+/src/build-id                       @umanwizard
+/src/build-info                     @benesch
+/src/ccsr                           @benesch
+/src/cloud-resources                @MaterializeInc/cloud
+/src/clusterd                       @MaterializeInc/compute @MaterializeInc/storage
+/src/compute                        @MaterializeInc/compute
+/src/compute-client                 @MaterializeInc/compute
+/src/environmentd                   @MaterializeInc/surfaces
+/src/expr                           @MaterializeInc/compute
+/src/expr-test-util                 @MaterializeInc/compute
+/src/frontegg-auth                  @MaterializeInc/compute
+/src/http-util                      @MaterializeInc/surfaces
+/src/interchange                    @MaterializeInc/storage
+/src/kafka-util                     @MaterializeInc/storage
+/src/kinesis-util                   @MaterializeInc/storage
+/src/lowertest                      @MaterializeInc/compute
+/src/lowertest-derive               @MaterializeInc/compute
+/src/metabase                       @benesch
+/src/mz                             @MaterializeInc/surfaces
+/src/npm                            @benesch
+/src/orchestrator                   @benesch
+/src/orchestrator-kubernetes        @benesch
+/src/orchestrator-process           @benesch
```

Comment:
Consider adding compute as an owner.

Reply:
For the process orchestrator specifically or all of them? (Leaving to a future PR btw.)